evernode-governor-hook/src/governor.c

```diff
@@ -851,6 +830,44 @@ int64_t hook(uint32_t reserved)
                 PERMIT_M("Evernode: Piloted mode candidate status changed.", vote_status);
             }
         }
+
+        if (redirect_op_type == OP_CHANGE_CONFIGURATION)
+        {
+            SET_UINT_STATE_VALUE(DEF_MOMENT_SIZE, CONF_MOMENT_SIZE, "Evernode: Could not initialize state for moment size.");
```

Comment:
But not setting the state. has the old value.

![image](https://user-images.githubusercontent.com/26274844/233323024-1e79cfb4-2df3-4dfd-84c2-b93ef1cf7c7b.png)